### ISSUE TITLE
UE4 Packaging Fails

### ISSUE THREAD

**Issue by freemancw** [ValveSoftware/steam-audio#24]
Packaging fails with:

```
error LNK2019: unresolved external symbol "unsigned int __cdecl SteamAudio::GetNumTrianglesForStaticMesh(class AStaticMeshActor *)" 

error LNK2019: unresolved external symbol "unsigned int __cdecl SteamAudio::GetNumTrianglesAtRoot(class AActor *)"
```

**Comment by Squareys** [comment]
Hi @freemancw !

Is this bug resolved? Is there a workaround or some other fix? I appear to have this problem also.

Kind regards,
Jonathan.

**Comment by freemancw** [comment]
Resolved in 4.17.
